app/src/main/res/values-fr/strings.xml

```diff
@@ -40,4 +40,18 @@
          so we put \u0020 to force a space. -->
     <string name="exclamation_point">\u0020!</string>
 
+
+    <string name="textview">TextView</string>
+    <string name="delete">Supprimer</string>
+    <string name="edit">Editer</string>
+    <string name="no_duplicate">"Les noms de quizzes ne peuvent pas être dupliquer"</string>
+    <string name="cancel">"Annuler"</string>
+    <string name="done">"Fini"</string>
+    <string name="yes">"Ouai"</string>
+    <string name="add_pop_up">"Rajouter un nom de quiz"</string>
+    <string name="error_blanck">"Cela ne peux pas ëtre blanc"</string>
+    <string name="edit_title">"Donner un nouveau nom"</string>
+    <string name="warning_delete">"Etes-vous sur de vouloir supprimer ce quiz, cela supprimera toutes les questions de ce quiz"</string>
```

Comment:
"Êtes-vous sûr de vouloir supprimer ce quiz? Cela va supprimer toutes les questions de ce quiz."

Reply:
Next time, I'll let you do it yourself if you wan't to change it